src/controllers/controlpickermenu.cpp

```diff
@@ -833,6 +833,18 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
             tr("Clears the search query"),
             pLibraryMenu);
 
+    // Color selection
+    addLibraryControl("track_color_next",
+            tr("Select Next Color Available"),
+            tr("Affect next color in the color palette"
+               " to the selected tracks"),
```

Comment:
Why "affect"?
```suggestion
            tr("Selects the next color in the color palette"
               " for the first selected track"),
```
It only changes the first selected track, see https://github.com/mixxxdj/mixxx/blob/08be88ff85f82e2ae573a554f0a29d64103b06cb/src/widget/wtracktableview.cpp#L439

Please change the pr strings here and for the deck below.